SignalServiceKit/src/Contacts/SignalRecipient.m

```diff
@@ -3,7 +3,9 @@
 //
 
 #import "SignalRecipient.h"
+#import "PhoneNumber.h"
```

Comment:
minor point, but I don't see any new code that uses these imports?

Reply:
Removed.